emission/net/api/stats.py

```diff
@@ -46,22 +46,33 @@ def storeClientEntry(user, key, ts, reading, metadata):
   currEntry = createEntry(user, key, ts, reading)
   # Add the os and app versions from the metadata dict
   currEntry.update(metadata)
-  get_client_stats_db().insert(currEntry)
+  response = get_client_stats_db().insert(currEntry)
```

Comment:
Add some sanity checks here. Having the instrumentation server go down or be misconfigured should not affect the main operation of the server. Adding a try/catch block over the entire method should be good enough, but you can also look at alternative strategies on the internet.

Please add a negative test case for this as well.